spec/02-integration/02-cmd/02-start_stop_spec.lua

```diff
@@ -342,6 +342,50 @@ describe("kong start/stop #" .. strategy, function()
           nginx_conf = "spec/fixtures/custom_nginx.template",
         }))
 
+        helpers.wait_until(function()
+          -- get a connection, retry until kong starts
+          helpers.wait_until(function()
+            local pok
+            pok, proxy_client = pcall(helpers.proxy_client)
+            return pok
+          end, 10)
+
+          local res = assert(proxy_client:send {
+            method = "GET",
+            path = "/",
+            headers = {
+              host = "example.test",
+            }
+          })
+          local ok = res.status == 200
+
+          if proxy_client then
+            proxy_client:close()
+            proxy_client = nil
+          end
+
+          return ok
+        end, 10)
+      end)
+      it("starts with a valid declarative config string", function()
+        local config_string = [[{"_format_version":"1.1","services":[{"name":"my-service","url":"http://127.0.0.1:15555","routes":[{"name":"example-route","hosts":["example.test"]}]}]}]]
+        local proxy_client
+
+        finally(function()
+          helpers.stop_kong(helpers.test_conf.prefix)
+          if proxy_client then
+            proxy_client:close()
+          end
+        end)
+
+        assert(helpers.start_kong({
+          database = "off",
+          declarative_config = nil,
+          declarative_config_string = config_string,
+          nginx_worker_processes = 100, -- stress test initialization
```

Comment:
I don't think this would "stress" kong much.

The "loading of the string" happens on the init phase, and that phase is run on a single thread. When workers are created afterwards, they "copy" the Lua state which was created on the init phase. This is usually very fast.

A more appropriate way to stress kong out would be to continuously update the declarative config via simultaneous calls to `kong reload` and `POST`s requests to the `/config` Admin API endpoint. There are locks in place (the `try_lock` calls) which should prevent kong from being overrun. It should just "reject updates while it is already updating".

But that is out of the scope of this PR I think.

TL;DR: I think we should just remove this line

```suggestion
```